include/kyosu/functions/reldist.hpp

```diff
@@ -18,10 +18,9 @@ namespace kyosu
   struct reldist_t : eve::strict_elementwise_callable<reldist_t, Options, eve::pedantic_option, eve::numeric_option>
   {
     template<concepts::cayley_dickson_like Z0, concepts::cayley_dickson_like Z1>
-    KYOSU_FORCEINLINE constexpr auto operator()(Z0 c0, Z1 c1) const noexcept -> decltype(kyosu::dist(c0, c1))
+    KYOSU_FORCEINLINE constexpr auto operator()(Z0 z0, Z1 z1) const noexcept -> decltype(kyosu::dist(z0, z1))
```

Comment:
dist has a proper return type, use it. This creates an unnecessary dependency.

Reply:
done